_content/tour/por/methods.article

```diff
@@ -43,67 +43,70 @@ and pointer receivers for implementing pointer semantics.
-the call matches the receiver’s data semantics exactly. The compiler just wants
-a value or pointer of the same type.
+Ao fazer uma chamada de método, o compilador não se importa se o valor usado
+para fazer a chamada corresponde exatamente à semântica dos dados do receiver.
+O compilador só quer um valor ou ponteiro do mesmo tipo.
 
     bill := user{"Bill", "bill@email.com"}
     bill.notify()
     bill.changeEmail("bill@hotmail.com")
 
-You can see that a value of type user is constructed and assigned to the bill
-variable. In the case of the notify call, the bill variable matches the receiver
-type which is using a value receiver. In the case of the changeEmail call, the
-bill variable doesn’t match the receiver type which is using a pointer receiver.
-However, the compiler accepts the method call and shares the bill variable with
-the method. Go will adjust to make the call.
+Você pode ver que um valor do tipo user é construído e atribuído à variável
+bill. No caso da chamada do notify, a variável bill corresponde ao tipo receiver
+que é um receiver de valor. No caso da chamada do changeEmail, a variável bill
+não corresponde ao tipo de receiver que é um receiver de ponteiro. Entretanto,
+o compilador aceita a chamada do método e compartilha a variável bill com o
+método. Go se ajustará para fazer a chamada.
 
-This works the same when the variable used to make the call is a pointer variable.
+Isso funciona da mesma forma quando a variável usada para fazer a chamada é uma
+variável de ponteiro.
 
     bill := &user{"Bill", "bill@email.com"}
     bill.notify()
     bill.changeEmail("bill@hotmail.com")
 
-In this case, the bill variable is a pointer variable to a value of type user.
-Once again, Go adjusts to make the method call when calling the notify method.
+Neste caso, a variável bill é uma variável de ponteiro para um valor do tipo user.
+Mais uma vez, Go se ajusta para fazer a chamada de método ao chamar o método notify.
 
-If Go didn’t adjust, then this is what you would have to do to make those same method calls. 
+Se Go não se ajustasse, então isso é o que você teria que fazer para realizar as mesmas
+chamadas de método.
 
     bill := user{"Bill", "bill@email.com"}
     (&bill).changeEmail("bill@hotmail.com")
 
     bill := &user{"Bill", "bill@email.com"}
     (*bill).notify()
 
-I’m glad you don’t have to do that to make method calls in Go.
+Fico feliz que você não precise fazer isso para fazer chamadas de método em Go.
 
-** Data Semantic Guideline For Internal Types
+** Guideline de Semântica de Dados Para Tipos Internos
 
-As a guideline, if the data I’m working with is an internal type (slice, map,
-channel, function, interface) then use value semantics to move the data around
-the program. This includes declaring fields on a type. However, when I’m reading
-and writing you need to remember I’m using pointer semantics.
+Como uma guideline, se os dados com os quais estou trabalhando forem um tipo
```

Comment:
I did a translation for guideline - Orientação. I'm happy if we do not translate. But then I need to update my PR.

Reply:
Yeah, I think Orientação is better, thanks!